guides/common/modules/con_understanding-package-mode-and-image-mode-hosts.adoc

```diff
@@ -0,0 +1,31 @@
+[id="understanding_package_mode_and_image_mode_hosts_{context}"]
+= Understanding Package mode and Image mode hosts
+
+In the latest update, hosts can now operate in two modes: package mode and image mode.
+
+.Image mode hosts
+Image mode hosts include eight new fields that reflect their boot status.
```

Comment:
```suggestion
Image mode hosts include additional fields that reflect their boot status.
```

I would not use "new" because it will get outdated & maybe also drop "eight" because it might become more/less fields in the future.